packages/relay/src/lib/eth.ts

```diff
@@ -1,8 +1,8 @@
-/* -
+/*-
  *
  * Hedera JSON RPC Relay
  *
- * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```

Comment:
nit: why change. Should revert the license changes

Reply:
Gotcha reverted it!